proposals/low-level-struct-improvements.md

```diff
@@ -173,6 +173,8 @@ ref struct ReadOnlyExample
 
 A `readonly ref struct` will require that `ref` fields are marked as `readonly ref`. There is no requirement that they are marked as `readonly ref readonly`. This does allow a `readonly struct` to have indirect mutations via such a field but that is no different than a `readonly` field that pointed to a reference type today.
 
+A `readonly ref` will be emitted to metadata using the `initonly` flag, same as any other field. A `ref readonly` field will be attributed with `System.Runtime.CompilerServices.IsReadOnlyAttribute`. A `readonly ref readonly` will be emitted with both items.
```

Comment:
It looks like this is consistent with how 'in' parameters are emitted and how 'ref readonly' parameters are planned to be emitted. Wanted to say this out loud so someone can correct me if I'm wrong. :)

Reply:
It's different from `in` in that `ref` fields will never have a `modreq`.